core-bundle/src/Resources/contao/classes/Backend.php

```diff
@@ -537,19 +539,17 @@ protected function getBackendModule($module, PickerInterface $picker = null)
 							$trail[] = ' <span>' . $GLOBALS['TL_LANG']['MOD'][$table] . '</span>';
 						}
 
+						$strUrl = $container->get('router')->generate('contao_backend', array
+						(
+							'do' => $container->get('request_stack')->getCurrentRequest()->query->get('do'),
+							'table' => $table,
+							'id' => $objRow->id,
+							'ref' => $container->get('request_stack')->getCurrentRequest()->attributes->get('_contao_referer_id'),
+							'rt' => REQUEST_TOKEN,
+						));
+
 						// Add object title or name
-						if ($objRow->title)
-						{
-							$trail[] = ' <span>' . $objRow->title . '</span>';
-						}
-						elseif ($objRow->name)
-						{
-							$trail[] = ' <span>' . $objRow->name . '</span>';
-						}
-						elseif ($objRow->headline)
-						{
-							$trail[] = ' <span>' . $objRow->headline . '</span>';
-						}
+						$trail[] = sprintf(' <span><a href="%s">%s</a></span>', $strUrl, ($objRow->title ?? $objRow->name ?? $objRow->headline));
```

Comment:
Will this always work or should we use `?:` instead of `??`?

Reply:
Ah right, we probably should also ignore empty values. Fixed in 1fa1fc05f8bdbe136fd99c9a4f8a132e9c94782f.